src/ORM/DatabaseAdmin.php

```diff
@@ -55,9 +59,22 @@ class DatabaseAdmin extends Controller
 
     /**
      * Config setting to enabled/disable the display of record counts on the dev/build output
+     * @deprecated 5.4.0 Will be replaced with SilverStripe\Dev\Command\DbBuild.show_record_counts
```

Comment:
```suggestion
```
Class is already deprecated and property is the same name

Reply:
Discussion can be in https://github.com/silverstripe/silverstripe-framework/pull/11365#discussion_r1755986594 given config is effectively the same as a static method.